python/mujinplanningclient/binpickingplanningclient.py

```diff
@@ -46,641 +75,2936 @@ def __init__(self, regionname=None, **kwargs):
+        if ioSignalsInfo is not None:
+            taskparameters['ioSignalsInfo'] = ioSignalsInfo
+        if isGripperSyncExecPossible != False:
+            taskparameters['isGripperSyncExecPossible'] = isGripperSyncExecPossible
+        if isTestMode != False:
+            taskparameters['isTestMode'] = isTestMode
+        if isStopOnGripperPositionNotReached != False:
+            taskparameters['isStopOnGripperPositionNotReached'] = isStopOnGripperPositionNotReached
+        if isStopOnObjectMassPropertiesMismatch != True:
+            taskparameters['isStopOnObjectMassPropertiesMismatch'] = isStopOnObjectMassPropertiesMismatch
+        if isStopOnPieceLost != True:
+            taskparameters['isStopOnPieceLost'] = isStopOnPieceLost
+        if isStopOnRobotExecutionError != False:
+            taskparameters['isStopOnRobotExecutionError'] = isStopOnRobotExecutionError
+        if isStopOnTorqueLimitsError != False:
+            taskparameters['isStopOnTorqueLimitsError'] = isStopOnTorqueLimitsError
+        if isStopOnTorqueLimitsErrorOutsidePickLocation != False:
+            taskparameters['isStopOnTorqueLimitsErrorOutsidePickLocation'] = isStopOnTorqueLimitsErrorOutsidePickLocation
+        if isStopOnControllerError != False:
+            taskparameters['isStopOnControllerError'] = isStopOnControllerError
+        if itlProgramNamesOnEvent is not None:
+            taskparameters['itlProgramNamesOnEvent'] = itlProgramNamesOnEvent
+        if cyclePreconditionIOInfo is not None:
+            taskparameters['cyclePreconditionIOInfo'] = cyclePreconditionIOInfo
+        if logmessagesmask != 0:
+            taskparameters['logmessagesmask'] = logmessagesmask
+        if jittererParameters is not None:
+            taskparameters['jittererParameters'] = jittererParameters
+        if justInTimeToolChangePlanning is not None:
+            taskparameters['justInTimeToolChangePlanning'] = justInTimeToolChangePlanning
+        if labelPlacingInfo is not None:
+            taskparameters['labelPlacingInfo'] = labelPlacingInfo
+        if labelerDirection is not None:
+            taskparameters['labelerDirection'] = labelerDirection
+        if localTargetDir is not None:
+            taskparameters['localTargetDir'] = localTargetDir
+        if logFailedTargetTimeout != 15:
+            taskparameters['logFailedTargetTimeout'] = logFailedTargetTimeout
+        if logFailedTargetPriorityTimeout != 10:
+            taskparameters['logFailedTargetPriorityTimeout'] = logFailedTargetPriorityTimeout
+        if maxAllowedTargetSize is not None:
+            taskparameters['maxAllowedTargetSize'] = maxAllowedTargetSize
+        if maxAllowedTargetSizeObjectName is not None:
+            taskparameters['maxAllowedTargetSizeObjectName'] = maxAllowedTargetSizeObjectName
+        if maxConsideredCameraIkSolutions != 200:
+            taskparameters['maxConsideredCameraIkSolutions'] = maxConsideredCameraIkSolutions
```

Comment:
This also says 200 in the schema, but is 20 in the server.

Reply:
I was told that something you can tell juniors during code review is "Imagine someone stumbling over this 2 years after you leave and being so annoyed that they hunt you down on Facebook", so they would write more defensively and hopefully double check their work. I'm not going to hunt Andriy down on Facebook, but git blame made me think of this story.

Reply:
(This _is_ public enough where he can find it :smile: .)

But, yeah, I was looking at many of these and wondering why we wanted a default.

Reply:
Fixed.